src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null || repeatPassword == null
+                || password.isEmpty() || repeatPassword.isEmpty()) {
+            throw new PasswordValidationException("Wrong passwords");
+        }
+        int riteLengthPassword = 10;
```

Comment:
let's make this number a constant and also create a more informative name for it